include/object/tcb.h

```diff
@@ -217,7 +217,7 @@ exception_t invokeTCB_NotificationControl(tcb_t *tcb, notification_t *ntfnPtr);
 cptr_t PURE getExtraCPtr(word_t *bufferPtr, word_t i);
 void setExtraBadge(word_t *bufferPtr, word_t badge, word_t i);
 
-exception_t lookupExtraCaps(tcb_t *thread, word_t *bufferPtr, seL4_MessageInfo_t info);
+exception_t lookupExtraCaps(tcb_t *thread, word_t *bufferPtr, word_t length);
```

Comment:
Naming sanity check, purely for pragmatics:

I'm right that `length` indeed happens to be the length of `bufferPtr`, right? Reading the loop starting at  `src/object/tcb.c:395`, this seems to be the case.

 I'd find it jarring to have a buffer and length argument, when length is not buffer's length.

Reply:
Nope, it's the length of the message, which determines if we need the ipc buffer or not.